### ISSUE TITLE
Toggle Button / Icon only - Lacks accessible name

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1235]
CENG_3317

Docs: https://react-magma.cengage.com/version/4.3.0/api/toggle-button/

Success Criteria: https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

The icon only toggle button has the correct attributes of aria-checked and role but lacks an accessible name. Because of this, users of assistive technology will hear the state of what they are toggling but not the name of it.

The icon only toggle button should have an accessible name added and this can be done in a few different ways. Options include using visually hidden text, an aria-label to name the button, or aria-labelledby to name the button based on text that is elsewhere on the page.

**Comment by chris-cedrone-cengage** [comment]
Verified aria-labels, all set!
